addons/im_livechat/static/src/embed/common/chatbot/chatbot_service.js

```diff
@@ -40,6 +43,14 @@ export class ChatBotService {
                 this.start();
             }
         });
+        this.livechatInitializedService.ready.then(() => {
+            this.busService.subscribe("im_livechat.bot_command", (payload) => {
+                this.store.insert(payload);
+                this.livechatService._saveLivechatState(payload.Thread, { persisted: true });
```

Comment:
Why do we need to save the live chat state? We only save id and model, which remain unchanged after a bot command if I'm not mistaken.

Reply:
It will update the operator and other cookie data in the localstorage.

Reply:
IMO, nothing needs to be updated:
- `threadData` is already known (id, model).
- `operator` is irrelevant, as it is used to connect with an actual operator. Changing the stored value with the new chat bot partner ID will actually prevent the user from reaching their preferred operator next time he opens a chat.